lib/flash_flow/version.rb

```diff
@@ -1,3 +1,3 @@
 module FlashFlow
-  VERSION = '1.4.7'
+  VERSION = '1.4.8'
```

Comment:
This is actually a version behind master now. Generally with the gem I've been bumping the version right on master, so I'd prefer we roll this back or it will overwrite the version now in master.